docs/api.md

```diff
@@ -1100,6 +1100,61 @@ PyWorker("worker.py", type="micropython")
 <div id="output"></div>  <!-- The display target -->
 ```
 
+!!! info
+
+    Sometimes code running on a worker gets stuck in an infinite loop and
+    becomes unresponsive. There are many complicated reasons why this might
+    happen, but when it does happen you likely want to break out of this
+    loop. This is where the `isStuck` and `notStuck` features come into play.
+
+    If you have code in the worker that could end up in an unresponsive
+    infinite loop, for example:
+
+    ```python
+    import time
+
+    while True:
+        time.sleep(1)
+        print("Stuck in a loop") 
+    ```
+
+    ...then you only need wrap this code in the worker like this:
+
+    ```python
+    import time
+    from pyscript import sync
+    is_stuck = sync.isStuck
+    break_loop = sync.notStuck
+
+    def is_not_stuck(condition):
+        if is_stuck():
```

Comment:
`if not condition and is_stuck()` is preferable in case condition is dynamic and, when this gets invoked, is `False` so that the `while` loop can get out without need to throw (i.e. within that second, the condition becomes `False`)